### PR TITLE
Add Support for Kerberized Hive 3

### PR DESCRIPTION
**Pull request by frankgh**
- Test against a static kerberized ambari server running HDFS 3.1.1 and Hive 3.1.0
- Add groups: multiClusterSecurity, proxySecurity for finer grain test grouping
- Remove `test_pxf_hdp2_secure_no_impersonation` and `test_pxf_hdp2_secure_with_impersonation` jobs from the pipeline as they were running a very basic test with Kerberos. These jobs are now replaced with `test_pxf_dataproc_multinode_with_impersonation` and `test_pxf_dataproc_multinode_no_impersonation`
- Add __UUID__ to the workingDirectory for hdfs

Co-authored-by: Francisco Guerrero <aguerrero@pivotal.io>
Co-authored-by: Alexander Denissov <adenissov@pivotal.io>

### CODE REVIEW

File: concourse/pipelines/templates/pxf-tpl.yml [hunk 1]
```diff
@@ -49,11 +49,12 @@ groups:
       - test_pxf_hdp2_ubuntu18
 {% endif %}
       - test_pxf_hdp2_multinode_gpdb
-      - test_pxf_secure_multinode_gpdb
+      - test_pxf_dataproc_multinode_with_impersonation
```
Comment (denalex) [inline concourse/pipelines/templates/pxf-tpl.yml]: does "dataproc" here means it is also and always "secure" ?

File: concourse/pipelines/templates/pxf-tpl.yml [hunk 2]
```diff
@@ -840,64 +836,27 @@ jobs:
       passed:
       - compile_pxf
       trigger: true
-    - get: gpdb-pxf-dev-centos6-hdp2-secure
-  - task: test_pxf_secure
-    file: pxf_src/concourse/tasks/test_pxf_secure.yml
-    image: gpdb-pxf-dev-centos6-hdp2-secure
-    privileged: true
-    attempts: 2
+    - get: gpdb-pxf-dev-centos6-hdp2-server
```
Comment (denalex) [inline concourse/pipelines/templates/pxf-tpl.yml]: do we really need server-based docker to run the tests ? if yes, we also have hdp3-based server docker image

Comment (frankgh) [reply concourse/pipelines/templates/pxf-tpl.yml]: we need the server based image for gpdb and singlecluster jars. I switched it to `gpdb-pxf-dev-centos7-hdp3-server`

File: concourse/pipelines/templates/pxf-tpl.yml (hunk 1 above)
Comment (frankgh) [reply concourse/pipelines/templates/pxf-tpl.yml]: added secure for this jobs

**Pull request merged**